tasks/discover-granules/schemas/config.json

```diff
@@ -90,6 +90,11 @@
       "description": "flag to set DiscoverGranules duplicate handling behavior",
       "default": "replace",
       "type": "string"
+    },
+    "concurrency": {
+      "description": "concurrency used when querying Cumulus API for already discovered granules",
+      "default": 3,
```

Comment:
Didn't we find out that the CMA schema enforcement actually doesn't set default values when none are provided? I think we need to apply this default in code and not in the schema definition

Reply:
Oh, I see that you already are enforcing it in the code. So do we need this default in the schema?

Reply:
I put in a default because the other values in this config also have a default, presumably just to document what those values are.